### DISCUSSION TITLE
Highcharts.stockChart:  1) calling conventions from nicegui and 2) documentation

### DISCUSSION THREAD

**Discussion by metaperl** [zauberzeug/nicegui#1750] [Ideas / Feature Requests]
Hello, there is support for calling highcharts `stockChart` from nicegui - https://github.com/zauberzeug/nicegui/issues/284

But my question is [it appears](https://github.com/zauberzeug/nicegui/discussions/948) that one must supply two arguments to a `chart`  method - `type` and `extras` instead of simply having a chart method... i.e.: currently you must do:

    chart = ui.chart( .... , type = 'stockChart', extras = ['stock'])

instead of 

    chart = ui.stock_chart( ... )

are the current calling conventions done to create a 1-to-1 mapping with the highcharts API?

Also: where is the documentation for `stockChart`?
1. I dont know where to look at [the highcharts site](https://www.highcharts.com/blog/products/stock/)
2. The way to call `stockChart` in NiceGUI is not documented either. There are only a few comments on the `chart` method with no link to a working example.

**Comment by falkoschindler** [comment]
The current documentation of `ui.chart` might be limited, but we can't cover all Highcharts features in our documentation. Therefore there is no dedicated section about Highcharts' stock charts.

Note that the element `ui.chart` is an entry point for _all_ Highcharts variants, since we didn't want to clutter the `ui` namespace with too many elements. The section [Chart with extra dependencies
](https://nicegui.io/documentation/chart#chart_with_extra_dependencies) shows how to create a different chart type.

**Reply by metaperl** [reply]
1. Kindly update the URL in your reply from http://127.0.0.1:8080/documentation/chart#chart_with_extra_dependencies to https://nicegui.io/documentation/chart#chart_with_extra_dependencies
2. The section [Chart with extra dependencies](https://nicegui.io/documentation/chart#chart_with_extra_dependencies) says `to use, e.g., Highcharts.stockChart instead, set the type property to "stockChart".`   ... but is `type` enough?  [it appears](https://github.com/zauberzeug/nicegui/discussions/948) that one must supply two arguments to a `chart`  method - `type` and `extras`

**Reply by falkoschindler** [reply]
Thanks for the hint; I updated the URL. 🙂
Have you tried adding the "stock" extra? Does it work?
What exactly do you propose? Do you expect the documentation to add a remark about the "stock" extra?

**Comment by falkoschindler** [comment]
I'll close this discussion because the feature request remains unclear.
